spec/requests/items_spec.rb

```diff
@@ -84,4 +106,28 @@
       end
     end
   end
+
+  describe 'POST /items' do
+    context 'when item post request attributes are valid' do
+      valid_attributes = {name: 'Asparagus', sell_in: 10, quality: 25}
+      before { post '/api/v1/items', params: valid_attributes }
+
+      it 'returns status code 201' do
+        expect(response).to have_http_status(201)
+      end
+    end
+
+    context 'when an invalid request' do
+      valid_attributes = {title: 'Lorem Ipsum'}
+      before { post '/api/v1/items', params: valid_attributes }
+
+      it 'returns status code 422' do
+        expect(response).to have_http_status(422)
+      end
+
+      it 'returns a failure message' do
+        expect(response.body).to match(/Validation failed: Name can't be blank/)
```

Comment:
Request for changes: You can test that the number of items in the database has not changed.